app/views/layouts/_header.html.erb

```diff
@@ -180,6 +180,12 @@
           if($('#chat').is(':visible') == true)
             c.hide() ;
       });
+      $('.nav-item.dropdown').on('show.bs.dropdown', function () {
+        $('.nav-item.dropdown').addClass("no-hover");
```

Comment:
Since you are already using single quotes, why did you switch to double quotes in `addClass` method?
The style needs to be consistent!